js/ui/components/clubhouse.js

```diff
@@ -112,6 +112,50 @@ var ClubhouseNotificationSource = new Lang.Class({
     },
 });
 
+var ClubhouseButton = new Lang.Class({
+    Name: 'ClubhouseButton',
+    Extends: St.Button,
+
+    _init: function(clubhouseProxy) {
+        this.parent();
+        this._clubhouseProxy = clubhouseProxy;
+
+        this.add_style_class_name('clubhouse-shell-button');
+
+        this._icon_size = 64
+        let icon = new St.Icon({icon_name: 'user-home-symbolic',
+                                icon_size: this._icon_size});
+        this.add_actor(icon);
+        this.reposition();
+
+        Main.overview.connect('shown', () => {
```

Comment:
If we're only calling in a method without parameters, the simplest syntax for these callbacks is
```
Main.overview.connect('shown', this.updateVisibility.bind(this));
```